src/index.js

```diff
@@ -40,9 +78,35 @@ class ToDoList {
       this.removeToDo(toDo);
       toDoApp.remove();
     });
+    const checkBox = toDoApp.querySelector('.checkbox');
+    checkBox.addEventListener('change', () => {
+      this.check(toDoApp);
+      if (toDo.completed === true) {
+        this.toDo.completed = false;
+        console.log(toDo);
+      } else {
+        this.toDo.completed = true;
+        console.log(toDo);
+      }
+    });
+
+    const toDoDescription = toDoApp.querySelector('.todo-description');
+    toDoDescription.addEventListener('dblclick', () => {
+      const editInput = document.createElement('input');
+      editInput.classList.add('edit-input');
+      editInput.value = toDo.description;
+      toDoDescription.replaceWith(editInput);
+      editInput.focus();
+
+      editInput.addEventListener('blur', () => {
+        toDo.description = editInput.value;
+        const newToDoElement = this.createToDo(toDo);
+        toDoApp.replaceWith(newToDoElement);
+      });
+    });
```

Comment:
- [ ] It is great that you have implemented the functionality for editing tasks but I am not able to update an existing todo 😨 Kindly fix this issue 👍 If it is working on your side then kindly record a video showing that your editing todo functionality is working 😊

Reply:
It is working you need to double click on it and when you edited the todo you can just click out side of the input field.

Reply:
<img width="624" alt="Screen Shot 2023-05-06 at 20 47 34" src="https://user-images.githubusercontent.com/101590645/236643902-483675cf-b6f7-438b-9f55-3e6127a6a231.png">
<img width="641" alt="Screen Shot 2023-05-06 at 20 47 55" src="https://user-images.githubusercontent.com/101590645/236643906-0ada9e0e-5897-492d-a0ed-0b9129a2122e.png">

Reply:
Ok then it might be an issue on my side. But I would also recommend the next reviewer to verify it.

Thanks for responding back dear. Happy Coding